Jamulus.pro

```diff
@@ -402,6 +402,7 @@ HEADERS += src/buffer.h \
     src/global.h \
     src/protocol.h \
     src/recorder/jamcontroller.h \
+    src/ThreadPool.h \
```

Comment:
Please keep to code style.  All source file names are lower case (or should be).